cellpack/autopack/upy/simularium/simularium_helper.py

```diff
@@ -1398,23 +1398,35 @@ def raycast_test(self, obj, start, end, length, **kw):
     def post_and_open_file(self, file_name, open_results_in_browser=True):
         simularium_file = Path(f"{file_name}.simularium")
         url = None
+        job_id = os.environ.get("AWS_BATCH_JOB_ID", None)
+        if job_id:
+            print(f"Batch Job ID: {job_id}")
         file_name, url = simulariumHelper.store_result_file(
-            simularium_file, storage="aws"
+            simularium_file, storage="aws", batch_job_id=job_id
         )
         if file_name and url:
-            simulariumHelper.store_metadata(file_name, url, db="firebase")
+            simulariumHelper.store_metadata(
+                file_name, url, db="firebase", job_id=job_id
+            )
             if open_results_in_browser:
                 simulariumHelper.open_in_simularium(url)
 
     @staticmethod
-    def store_result_file(file_path, storage=None):
+    def store_result_file(file_path, storage=None, batch_job_id=None):
         if storage == "aws":
             handler = DATABASE_IDS.handlers().get(storage)
-            initialized_handler = handler(
-                bucket_name="cellpack-results",
-                sub_folder_name="simularium",
-                region_name="us-west-2",
-            )
+            if batch_job_id:  # save results to batch job bucket
+                initialized_handler = handler(
+                    bucket_name="cellpack-demo",
```

Comment:
the temporary bucket name will be updated later

Reply:
May be worth adding a comment explaining what this is and when we'll be able to remove it? Like
```
\\ if there's a batch_job_id, this is running in AWS batch, and we should use the temp bucket
\\ this can be removed once the AWS batch job has access to the cellpack-results bucket
```